src/docs/deployments/setup/DOC.md

```diff
@@ -59,3 +59,8 @@ Travis-CI should be enabled for your dev repo
 13. Confirm the build completed successfully on Travis-CI and verify that the files were pushed to your build repo’s ci-beta branch.
 
 14. For a description of which branches route to which environments, please see the [insights-frontend-starter-app README.md](https://github.com/RedHatInsights/insights-frontend-starter-app/)
+
+## Configuring Your App in cloud-services-config
```

Comment:
This needs to be surrounded by blank lines
```
src/docs/deployments/setup/DOC.md: 63: MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] [Context: "## Configuring Your App in cloud-services-config"]
```

Travis is going to fail this for linting 😄

Reply:
Oops! Didn't give me a linting error locally. I'll fix that